openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -22,7 +22,7 @@ <h2 class="staff-info__title ui-staff__title">
                 <div class="staff-info__student">
                     <div class="wrapper--input" class="staff-info__student__form">
                         <form class="openassessment_student_info_form">
-                            <div class="form--error"></div>
+                            <div class="form--error" tabindex="-1"></div>
```

Comment:
Note to reviewers: -1 allows focus to be moved to item programmatically, but keeps the item out of the normal tab ordering.